cypress/e2e/environment.cy.js

```diff
@@ -0,0 +1,389 @@
+const newProjectName = '[aria-label="Project Name-select-input';
+const newProjectNameButton = ".pf-c-select__menu-item";
+const nameInput = '[aria-label="Name-input"]';
+const descInput = '[aria-label="Description-input"]';
+const repoInput = '[aria-label="Repository-input"]';
+const branchInput = '[aria-label="Branch-input"]';
+const iconInput = "#simple-text-file-filename";
+
+const testProjectName = (number) => "Test Project Name " + number;
+const testName = (number) => "TestName " + number;
+
+const createEnv = (number, shouldTypeName = true, isShort = false) => {
+  cy.get(projectNameInput).type(testProjectName(number));
+  cy.get(newProjectName).within(() => {
+    cy.get(newProjectNameButton).click();
+  });
+  if (shouldTypeName) {
+    cy.get(nameInput).type(testName(number));
+  }
+  if (!isShort) {
+    cy.get(descInput).type("Test description");
+    cy.get(repoInput).type("repository");
+    cy.get(branchInput).type("branch");
+    cy.get(iconInput).attachFile(icon);
+  }
+};
+
+const deleteEnv = (number) => {
+  cy.get("button").contains("Delete environment").click();
+  cy.get('[aria-label="delete"]').should("be.disabled");
+
+  cy.get('[aria-label="delete environment check"]').type(testName(number));
+  cy.get('[aria-label="delete"]').should("not.be.disabled").click();
+  cy.url().should("eq", Cypress.config().baseUrl + "/console/");
+  cy.get('[aria-label="Environment card"]')
+    .contains(testProjectName(number))
+    .should("not.exist");
+};
+
+describe("Environment", function () {
+  it("1.1 cancel creation of an environment", function () {
+    cy.visit("/console/");
+    cy.get(newEnvCard).click();
+    cy.url().should(
+      "eq",
+      Cypress.config().baseUrl + "/console/environment/create"
+    );
+    createEnv(1);
+    cy.get("button").contains("Cancel").click();
+    cy.url().should("eq", Cypress.config().baseUrl + "/console/");
+  });
+
+  it("1.2 Create ne environment", function () {
+    //fill the form and submit
+    cy.visit("/console/environment/create");
+    createEnv(2, false);
+    cy.get("button").contains("Submit").should("be.disabled");
+    cy.get(nameInput).type(testName(2));
+    cy.get("button").contains("Submit").should("not.be.disabled");
+    cy.get("button").contains("Submit").click();
+    cy.url().should("contain", "/console/lsm/catalog?env=");
+    //go back to gome and check if env is visible
+    cy.get(".pf-c-breadcrumb__item").first().click();
+    cy.url().should("eq", Cypress.config().baseUrl + "/console/");
+    cy.get('[aria-label="Environment card"]').should(
+      "any.contain",
+      testName(2)
+    );
+  });
+
+  it("1.3 delete an environment", function () {
+    //Fill The form and submit
+    cy.visit("/console/environment/create");
+    createEnv(3, true, true);
+    cy.get("button").contains("Submit").click();
+    cy.url().should("contain", "/console/lsm/catalog?env=");
+    //Go to settings
+    cy.get('[aria-label="Settings actions"]').click();
```

Comment:
Go to settings is being reused in a few places too, handy to have it a as a method